### PR TITLE
Minor NAD integration improvements

### PR DESCRIPTION
**Pull request by atj**
## Proposed change

This PR includes some minor improvements to the NAD integration documentation.

It also includes a change to the valid range of the `sources` configuration variable. Please see the following core PR:

https://github.com/home-assistant/core/pull/72086

## Type of change

- [x] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase:
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by homeassistant** [comment]
Hi @atj, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;home-assistant.io#22804).

Once you do that we will be able to review and accept this pull request.

Thanks!

**Comment by atj** [comment]
> It seems that this PR is targeted against an incorrect branch since it has a parent PR on one of our codebases. Documentation that needs to be updated for an upcoming release should target the `next` branch. Please change the target branch of this PR to `next` and rebase if needed. (message by DocsTargetBranch)

Please ignore this, I misunderstood what "Link to parent pull request in the codebase" meant - I've now removed the link to the core PR.

File: source/_integrations/nad.markdown [hunk 1]
```diff
@@ -71,7 +79,7 @@ max_volume:
   default: -20
   type: integer
 sources:
-  description: A list of mappings from source to source name. Valid sources are `1 to 10`. (for `RS232` and `Telnet` types)
+  description: A list of mappings from source to source name. Valid sources are `1 to 12`. (for `RS232` and `Telnet` types)
```
Comment (MartinHjelmare) [inline source/_integrations/nad.markdown:82]: If we're changing the source range we should target next branch since that requires the parent PR in core.

Comment (atj) [reply source/_integrations/nad.markdown:82]: Thanks, I've now changed the PR to the next branch.

**Comment by atj** [comment]
Apologies for the noise everyone - I tried to change the targeted branch in git rather than using the feature in GitHub, which resulted in a load of unrelated commits being added to the PR.

**Review by MartinHjelmare — APPROVED** [review]
Thanks!

**Pull request merged**